### PR TITLE
Crimmeria Fix Byz requirement change Hellenic Kingdom Rework

### PR DESCRIPTION
**Pull request by JakeP12**
Crimmeria Fix Byz requirement change Hellenic Kingdom Nerf

### CODE REVIEW

**Review by IsaacBalcells — COMMENTED** [review]
I have no comments, but seem a bit adhoc for the player playing there :P

**Comment by JakeP12** [comment]
> I have no comments, but seem a bit adhoc for the player playing there :P

True i would've not fix the crimmeria fornable if i wasnt playing there lol but in my next commit i will take out the export buff and replace it with something shit lol

File: common/named_colors/formable_MolonLabe_colors.txt [hunk 1]
```diff
@@ -197,7 +197,7 @@ colors = {
 
 	liguria_color = rgb { 157 75 26 }
 	
-	Cimmeria_color = rgb { 173 0 40 }
+	Cimmeria_color = rgb { 13 252 148 }
```
Comment (x3igh7) [inline common/named_colors/formable_MolonLabe_colors.txt:200]: would love to see these properly cased to reduce likelihood of errors

File: events/formable_heritage_event.txt [hunk 2]
```diff
@@ -5768,4 +5768,48 @@ formable_heritage.150 = {
...
+    }	
+    option = { # EFFECT FIELD - THIRD OPTION
+        name = formable_heritage.180.c # LOCALIZATION KEY - Option B text
+		hidden_effect = { 
+			set_country_heritage = tribal_heritage 
+		}
+		show_as_tooltip = {
+			add_country_modifier = {
+				name = tribal_modifier
+				#duration = -1
```
Comment (x3igh7) [inline events/formable_heritage_event.txt:5811]: just delete this line if it's not being used

File: events/formable_heritage_event.txt [hunk 3]
```diff
@@ -5768,4 +5768,48 @@ formable_heritage.150 = {
...
+
+    option = { # EFFECT FIELD - FIRST OPTION
+        name = formable_heritage.180.a # LOCALIZATION KEY - Option A text
+        hidden_effect = { 
+			set_country_heritage = Cimmeria_heritage
+		} # EFFECT
+		show_as_tooltip = {
+			add_country_modifier = {
+				name = Cimmeria_modifier
+				#duration = -1
```
Comment (x3igh7) [inline events/formable_heritage_event.txt:5795]: just delete this line if it's not being used

**Comment by x3igh7** [comment]
couple comment, think there is 1 error in there that should be with the AI chance, probably just an accidental keystroke

**Comment by JakeP12** [comment]
> couple comment, think there is 1 error in there that should be with the AI chance, probably just an accidental keystroke

Yeah the F was by mistake lol when click to search 
i didnt take out those lines just keep them the same as everything else

File: common/named_colors/formable_MolonLabe_colors.txt (hunk 1 above)
Comment (JakeP12) [reply common/named_colors/formable_MolonLabe_colors.txt:200]: There is no errors they work fine in game

**Comment by JakeP12** [comment]
> > couple comment, think there is 1 error in there that should be with the AI chance, probably just an accidental keystroke
> 
> Yeah the F was by mistake lol when click to search i didnt take out those lines just keep them the same as everything else

I've taken the F out so once you approve this I will do another pull that will take it out

**Comment by JakeP12** [comment]
> > > couple comment, think there is 1 error in there that should be with the AI chance, probably just an accidental keystroke
> > 
> > 
> > Yeah the F was by mistake lol when click to search i didnt take out those lines just keep them the same as everything else
> 
> I've taken the F out so once you approve this I will do another pull that will take it out

but yeah I'm not deleting those lines that code has been used over 6000 times no need to change and maybe break it lol

**Review by x3igh7 — APPROVED** [review]

**Pull request merged**